src/librustdoc/html/static/main.js

```diff
@@ -2157,6 +2157,13 @@ function defocusSearchBar() {
                 var code = document.createElement("code");
                 code.innerHTML = struct.text;
 
+                onEachLazy(code.getElementsByTagName("a"), function(elem) {
+                    var href = elem.getAttribute("href");
+
+                    if (href && href.indexOf("http") !== 0) {
+                        elem.setAttribute("href", rootPath + href);
+                    }
+                });
```

Comment:
This is newly added. What is it for?

Reply:
I somehow didn't remove the equivalent code (which is still just under). Fixing it.